src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +55,48 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        try {
+            if (dateParams.length == 0) {
+                return Optional.empty();
+            }
+            return  Optional.ofNullable(LocalDate.parse(dateParams[0].toString()
```

Comment:
we can simplify:
return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));